osmaxx/api_client/API_client.py

```diff
@@ -15,7 +15,6 @@ class RESTApiClient:
 
     def __init__(self, service_base):
         self.service_base = service_base
-        self.client = requests.session()
```

Comment:
As long as we're not using this session, we can indeed delete this code. But maybe we _should_ use a [session](http://docs.python-requests.org/en/master/api/#request-sessions)? We don't need cookie persistence, but connection-pooling might be good.

And it seems that `requests.Session` already can do what we implemented ourselves with `_data_dict()` and `_default_headers()`: http://docs.python-requests.org/en/master/api/#requests.Session.headers